src/templates/app/user/security/cloudflare_dns.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const CloudflareDNS = () => (
+    <React.Fragment>
+        <h1>1.1.1.1</h1>
+        <p>{it.L('Cloudflare\'s public DNS service gives you more secure and faster web browsing.')}</p>
+        <div className='gr-row gr-row-align-center gr-padding-10'>
+            <div className='gr-7 gr-12-m'>
+                <img className='responsive' src={it.url_for('images/pages/settings/connect-1111.svg')} />
+            </div>
+        </div>
+        <div className='gr-row'>
+            <div className='gr-12 gr-padding-10'>
+                <h2>{it.L('What is DNS')}</h2>
+                <p>{it.L('The Domain Name System (DNS) allows you to access a website through a user-friendly web address such as [_1], instead of a complicated IP address.', `<a href="${it.url_for('/')}">www.binary.com</a>`)}</p>
+            </div>
+            <div className='gr-12 gr-padding-10'>
+                <h2>{it.L('Why you should use 1.1.1.1 instead of your ISP\'s default DNS service')}</h2>
```

Comment:
if we're not translating 1.1.1.1 in other strings, we should also use a placeholder to not translate it here